Enterprise-CMCS/macpro-mdct-qmr
# IU-HH Text Change 2382

### Description
Just a small text change for this measure


### Related ticket(s)
MDCT-2382

---
### How to test
1. Log in to a state with Health Home measures (for example, `stateuser6`).
2. Select FFY 2023.
3. Navigate to this measure.
4. Fill out the first three questions, to get the performance measure text to appear.
5. Check that text.

### Important updates
n/a

---
### Author checklist
